1-mandatory/3-slideshow/index.html

```diff
@@ -12,6 +12,11 @@
   </head>
   <body>
     <!-- Write your HTML in here -->
+    <div class="wrap">
+      <button id="btn1">Next</button>
```

Comment:
Small design comment: I would expect to see the Next button on the right and Previous on the left, rather than the other way around like you have here.